RedfishPkg/RedfishHttpDxe/RedfishHttpOp.h

```diff
@@ -0,0 +1,75 @@
+/** @file
+  Definitions of RedfishHttpOp

```

Comment:
What the RedfishHttpOp refers? We may want to give it an understandable naming.

Reply:
That is Redfish Http Operation functions. Does RedfishHttpOperation.h sound good?

Reply:
Yes,  RedfishHttpOperation.h is good.